scripts/ci/download_gps_traces.py

```diff
@@ -0,0 +1,91 @@
+import requests
+import xml.etree.ElementTree as ET
+import csv
+import sys
+import argparse
+
+def get_osm_gps_traces(min_lon, min_lat, max_lon, max_lat):
```

Comment:
Downloads GPS traces from OSM in order to use them in benchmark: to make it a bit more realistic.

Reply:
Are these random traces each time or the same?

Reply:
Well, honestly I haven't checked, but I guess they are not guaranteed to be the same: if someone uploads new trace it may change output of this OSM API. WDYT? We probably could cache them on CI to make it more stable.

Reply:
I guess there is value in having stability. And catching the test data on CI may also make us resilient against a third-party network outage.

Reply:
Well, I see 2 options which can be done relatively easy:
- store set of these traces directly in repo
- just cache this data on CI. Here we have a risk that from time to time it will get updated since there is no guarantee that CI cache will always remain the same. But probably it is not a big problem.


WDYT?

Reply:
How big is the data set?

Reply:
The size of dataset is ~500kb if it is CSV compressed with gzip (we can easily decompress it on CI). So I think we can easily just add it to the repo. :+1: